### PR TITLE
Gra 1044 verbosity persistance

### PR DESCRIPTION
**Pull request by mattkasun**
also restructures netclient startup to maximize utility of cobra/viper libs

### CODE REVIEW

File: cmd/gui.go [hunk 1]
```diff
@@ -1,22 +1,38 @@
+//go:build !headless
+// +build !headless
+
+//go:generate goversioninfo -icon=resources/windows/netclient.ico -manifest=resources/windows/netclient.exe.manifest.xml -64=true -o=netclient.syso
```
Comment (0xdcarns) [inline cmd/gui.go:4]: I believe this has to stay in main

**Review by 0xdcarns — CHANGES_REQUESTED** [review]
address static checks as well

**Review by theguy951357 — COMMENTED** [review]
will wait on changes

**Comment by theguy951357** [comment]
getting an error on install 

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xa46bf8]

goroutine 1 [running]:
github.com/gravitl/netclient/cmd.initConfig()
        /root/netclient/cmd/root.go:67 +0xb8
github.com/spf13/cobra.(*Command).preRun(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:946
github.com/spf13/cobra.(*Command).execute(0x1183860, {0x1207140, 0x0, 0x0})
        /root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:876 +0x563
github.com/spf13/cobra.(*Command).ExecuteC(0x1182720)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/gravitl/netclient/cmd.Execute()
        /root/netclient/cmd/root.go:40 +0x25
main.main()
        /root/netclient/main.go:24 +0x4a

**Comment by theguy951357** [comment]
every command is doing this

root@ubuntuQA:~# netclient list
[netclient] 2023-01-27 16:06:56 setting OS 
[netclient] 2023-01-27 16:06:56 setting version 
[netclient] 2023-01-27 16:06:56 setting netclient hostid 
[netclient] 2023-01-27 16:06:56 setting name 
[netclient] 2023-01-27 16:06:56 setting macAddress 
[netclient] 2023-01-27 16:06:56 setting wireguard keys 
[netclient] 2023-01-27 16:06:56 setting traffic keys 

no such network

it is causing a hostid mismatch when I join a network.

**Review by theguy951357 — APPROVED** [review]
joined network with two nodes

changed verbosity with -v 2

verbosity in netclient.yml is set to 2

changed verbosity in UI to 4

verbosity in netclient yaml is 4

pass

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**